### PR TITLE
[WIP] Automatically create git tag for new versions

### PR DESCRIPTION
**Pull request by julianoes**
Addresses #541.

@JonasVautherin should we also do the merge with master automatically?

### CODE REVIEW

**Comment by julianoes** [comment]
So why on Earth did travis to decide to run this PR?

File: version [hunk 1]
```diff
@@ -1 +1 @@
-0.2.8
+0.6.0
```
Comment (JonasVautherin) [inline version]: Why 0.6.0?

**Comment by julianoes** [comment]
Let's see if this works in a dry-run.

**Comment by julianoes** [comment]
Unfortunately this didn't trigger:
https://travis-ci.org/Dronecode/DronecodeSDK/jobs/436987724#L9751

**Pull request merged**